HLTrigger/Configuration/scripts/hltCheckPrescaleModules

```diff
@@ -17,7 +15,12 @@ if len(sys.argv) != 2:
   sys.exit(1)
 
 # whitelist paths exempt from validation
-whitelist = ('HLTriggerFirstPath', 'HLTriggerFinalPath')
+whitelist = [
+  'HLTriggerFirstPath',
+  'HLTriggerFinalPath',
+  'Status_OnCPU',
+  'Status_OnGPU',
```

Comment:
This removes these expected errors:
```
ERROR: path Status_OnCPU dos not have an HLTPrescaler module
ERROR: path Status_OnGPU dos not have an HLTPrescaler module
```
https://cmssdt.cern.ch/SDT/jenkins-artifacts/HLT-Validation/CMSSW_12_6_X_2022-11-05-1100/el8_amd64_gcc10/HLT_Integration_GRun_MC.log